src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -232,6 +233,63 @@ namespace AirflowNetworkBalanceManager {
 	Real64
 	GetZoneInfilAirChangeRate( int const ZoneNum ); // hybrid ventilation system controlled zone number
 
+	// derived class or struct
+	struct OccupantVentilationControlProp {
+
```

Comment:
@jasondegraw You made good suggestions. However, I may have conversion issue. For example, I made a line like this:
OpeningStatus = OpenStatus::FreeOperation;

The compiler showed an error since OpeningStatus is defined as integer.
If I made some changes as shown below:

OpeningStatus = int( OpenStatus::FreeOperation );

The complier did not complain. Please show me examples and let me know which way is OK to go.